### ISSUE TITLE
Functionality of adding url instead of Code in local directory.

### ISSUE THREAD

**Issue by MohammedAlamkhan** [bridgelauncher/launcher#1]
I am working on windows phone aesthetics using html/css but everytime I need to delete old code from phone and paste new code.
I am using GitHub pages to host the web version, it'll be very helpful if I could just add my GitHub pages link and with every push to my repo my launcher will get updated.

**Comment by theothertored** [comment]
Great idea, added to my TODO list. Shouldn't be too hard to implement, though I've been incredibly busy recently.

**Comment by theothertored** [comment]
I will add that for now I suggest doing the majority of your testing on desktop by exporting your app list and using it with the API mock, if you're not doing that already. You could also set up a script to copy your files to your phone as for example a build action in VSCode or another editor.

**Comment by MohammedAlamkhan** [comment]
Thanks!! Mock API seems like the best way of development going forward.

**Comment by theothertored** [comment]
Yup, the exact idea behind the app list + icons export and the API mock is to allow development in a desktop environment without having to test every change on the actual device.

I will keep this issue open because loading from URL would make for a neat feature, be it to use your own projects and auto update, or to quickly try out someone else's project without having to download the files and point to the folder.
